app/uk/gov/hmrc/statepension/domain/nps/NpsSummary.scala

```diff
@@ -97,8 +97,12 @@ case class NpsAmountA2016(
                            post88COD: BigDecimal = 0,
                            grb: BigDecimal = 0
```

Comment:
You should change grb to graduatedRetirementBenefit there is no need to have two values the same.